scripts/refresh_box_tokens.py

```diff
@@ -44,7 +46,7 @@ def main(delta, dry_run):
     try:
         days = int(sys.argv[2])
     except (IndexError, ValueError, TypeError):
-        days = 60 - 7  # refresh tokens are good for 60 days
+        days = 7  # refresh tokens that expire this week
```

Comment:
It was previously set to refresh any tokens that expire at or before 53 days ago each time it was run. It's now set to refresh any tokens that expire at or before one week from the time it's run.